NodeBase/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM selenium/base:2.52.0
+FROM selenium/base:latest
```

Comment:
We'll merge it into master and after that into the 2.53.0 branch. Not using a specific tag here, will likely cause trouble later on.